src/sagemaker/workflow/airflow.py

```diff
@@ -0,0 +1,137 @@
+# Copyright 2017-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+import os
+
+from sagemaker.estimator import Framework
+from sagemaker.amazon.amazon_estimator import AmazonAlgorithmEstimatorBase, RecordSet
+from sagemaker.job import _Job
+from sagemaker.utils import base_name_from_image, airflow_name_from_base
+from sagemaker.model import DIR_PARAM_NAME, SCRIPT_PARAM_NAME, CLOUDWATCH_METRICS_PARAM_NAME, \
+    CONTAINER_LOG_LEVEL_PARAM_NAME, JOB_NAME_PARAM_NAME, SAGEMAKER_REGION_PARAM_NAME
```

Comment:
You should import modules not classes https://github.com/google/styleguide/blob/gh-pages/pyguide.md#224-decision

```python
from sagemaker.amazon import amazon_estimator
from sagemaker import model
...
```

Reply:
Cool. Updated for both airflow.py and test_airflow.py.